pages/windows/choco-push.md

```diff
@@ -0,0 +1,12 @@
+# choco-push
+
+> Push a compiled nupkg to a package feed.
+> More information: <https://docs.chocolatey.org/en-us/create/commands/push>.
+
+- Push a compiled nupkg to the specified feed:
+
+`choco push --source {{https://push.chocolatey.org/}}`
+
+- Push a compiled nupkg to the specified feed with a timeout in seconds (default is 2700):
+
+`choco push --source {{"'https://push.chocolatey.org/'"}} --execution-timeout {{500}}`
```

Comment:
Why are there `"` and `'`? Is this required for this command? Otherwise I suppose we could do 
```suggestion
`choco push --source {{"https://push.chocolatey.org/"}} --execution-timeout {{500}}`
```

or even 

```suggestion
`choco push --source {{https://push.chocolatey.org/}} --execution-timeout {{500}}`
```

Reply:
I'm not familiar on how quoting works on windows, so I can't really help :/

Reply:
> Maybe @kbdharun could help?

Sure, I will check it right now and get back.

Reply:
**Update:**

The commands work with or without the quotes too.

```ps
PS C:\Users\kbdha> choco
Chocolatey v2.1.0
Please run 'choco -?' or 'choco <command> -?' for help menu.
PS C:\Users\kbdha> choco push --source "'https://push.chocolatey.org/'" --execution-timeout 500
Chocolatey v2.1.0
An API key was not found for 'https://push.chocolatey.org/'. You must either set an API key with the apikey command or specify one with --api-key.
PS C:\Users\kbdha> choco push --source "https://push.chocolatey.org/" --execution-timeout 500
Chocolatey v2.1.0
An API key was not found for 'https://push.chocolatey.org/'. You must either set an API key with the apikey command or specify one with --api-key.
PS C:\Users\kbdha> choco push --source https://push.chocolatey.org/ --execution-timeout 500
Chocolatey v2.1.0
An API key was not found for 'https://push.chocolatey.org/'. You must either set an API key with the apikey command or specify one with --api-key.
```

Reply:
> So we can leave out the " and '? That would be better IMO

Yep, already applied the suggestion for it.